lib/api/client.dart

```diff
@@ -16,13 +16,13 @@ class Client {
   GlobalKey<ScaffoldMessengerState> global;
   final JsonDecoder _decoder = new JsonDecoder();
   final JsonEncoder _encoder = new JsonEncoder();
-  String? _token;
-  String? _base;
+  String _token = '';
+  String _base = '';
```

Comment:
This and other changes like this (`String?` --> `String`) were made because a nullable String, when interpolated, prints null instead of an empty string. Ex: `String? string; '$string' == 'null'`

Reply:
Ah, makes sense. I was wondering why you would change that!